src/core_plugins/kibana/public/visualize/editor/agg_params.js

```diff
@@ -138,7 +138,7 @@ uiModules
       }
 
       function getIndexedFields(param) {
-        let fields = $scope.agg.vis.indexPattern.fields.raw;
+        let fields = _.filter($scope.agg.vis.indexPattern.fields.raw, 'aggregatable');
```

Comment:
I'm guessing that this change may have caused scripted fields to no longer show in the list for metric aggregations?
https://github.com/elastic/kibana/issues/8585